pom.xml

```diff
@@ -84,6 +84,53 @@
                 <artifactId>biz.aQute.bndlib</artifactId>
                 <version>5.2.0</version>
             </dependency>
+
+            <!-- 3.0 used by tycho. Force the target maven version here. -->
+            <dependency>
+                <groupId>org.apache.maven</groupId>
+                <artifactId>maven-compat</artifactId>
+                <version>${maven.version}</version>
+            </dependency>
+            <!-- 3.0 used by tycho. Force the target maven version here. -->
+            <dependency>
+                <groupId>org.apache.maven</groupId>
+                <artifactId>maven-core</artifactId>
+                <version>${maven.version}</version>
+                <scope>provided</scope>
+            </dependency>
+            <!-- 3.0 used by tycho. Force the target maven version here. -->
+            <dependency>
+                <groupId>org.apache.maven</groupId>
+                <artifactId>maven-settings</artifactId>
+                <version>${maven.version}</version>
+                <scope>provided</scope>
+            </dependency>
+            <!-- 3.0 used by org.twdata.maven:mojo-executor. -->
+            <dependency>
+                <groupId>org.apache.maven</groupId>
+                <artifactId>maven-model</artifactId>
+                <version>${maven.version}</version>
+            </dependency>
+            <!-- 3.0 used by tycho -->
+            <dependency>
+                <groupId>org.apache.maven</groupId>
+                <artifactId>maven-plugin-api</artifactId>
+                <version>${maven.version}</version>
+            </dependency>
+            <!-- 3.0 used by maven-plugin-annotations -->
+            <dependency>
+                <groupId>org.apache.maven</groupId>
+                <artifactId>maven-artifact</artifactId>
+                <version>${maven.version}</version>
+            </dependency>
+
+            <!-- 1.5.5 used by tycho -->
```

Comment:
Here 1.5.5 is documented but 1.6 is used. What version is the right one?

Reply:
comment should be saying: 1.5.5 is used by tycho, other dependencies have 1.6 as transitive dependency. I will expand the comment

Reply:
Thank you for clarifying